.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,10 @@
+---
+name: "Bug report"
+about: Create a report to help improve the project
+labels: task/bug
```

Comment:
We use the `task/bug` label as our own triaging system (i.e. we only label as issue as `task/bug` once we confirm that it is indeed a bug) so we'd prefer if this didn't automatically add that label.

Reply:
Got it. I'll remove the automatic labels from both templates.